tests/baselines/reference/tsbuild/outFile/rebuilds-completely-when-version-in-tsbuildinfo-doesnt-match-ts-version.js

```diff
@@ -207,22 +207,22 @@ c.doSomething();
 
 Output::
 /lib/tsc --b /src/third --verbose
-12:00:00 AM - Projects in this build: 
+[[90m12:00:00 AM[0m] Projects in this build: 

```

Comment:
did these intentionally swap to `pretty` output?

Reply:
Yes.. The helper does that.